x/tokenfactory/keeper/createdenom_test.go

```diff
@@ -13,6 +13,9 @@ func (suite *KeeperTestSuite) TestMsgCreateDenom() {
 	suite.Setup()
 
 	// Creating a denom should work
+	senderAddress := suite.ChainA.SenderAccounts[0].SenderAccount.GetAddress()
+	suite.TopUpWallet(suite.ChainA.GetContext(), senderAddress, suite.TestAccs[0])
```

Comment:
shouldn't we also make sure that the creation fee is charged and received by the fee collector?